src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -217,7 +224,10 @@ def setup(self) -> None:
                 self.set_failed(pool)
                 return
 
-            logging.info("creating scaleset: %s", self.scaleset_id)
+            logging.info(
+                SCALESET_LOG_PREFIX + "creating scaleset scalset_id:%s",
```

Comment:
```suggestion
                SCALESET_LOG_PREFIX + "creating scaleset. scaleset_id:%s",
```

Reply:
You change convention in these messages wrt initial capitalization, and the boundary between the context message and the key/value data. Consider doing a pass and making it the same throughout the touched code.